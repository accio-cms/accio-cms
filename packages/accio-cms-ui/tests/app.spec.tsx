import React = require('react')
import { create } from 'react-test-renderer'

import App from '../src/app'

test("Say my name, say my name...", () => {
  const tree1 = create(
    <App />
  ).toJSON()
  expect(tree1).toMatchSnapshot()
})
