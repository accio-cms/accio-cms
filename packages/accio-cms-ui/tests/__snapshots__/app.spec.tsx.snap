// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Say my name, say my name... 1`] = `
<h1>
  POOP
</h1>
`;
